src/views/Timesheets/index.js

```diff
@@ -168,10 +168,10 @@ const Timesheets = props => {
   if (props.Authentication) {
     const getActiveJobsForUser = (dateIn, dateOut) => {
       let details = {
-        shift_start_datetime: dateIn.toLocaleString(),
-        shift_end_datetime: dateOut.toLocaleString(),
+        shift_start_datetime: dateIn.toISOString(),
+        shift_end_datetime: dateOut.toISOString(),
```

Comment:
Did you do this to replace a deprecated function? Just wondering.

Reply:
This is also a merge from develop, not a change by me